packages/tools/client-debugger/client-debugger/src/FluidDevtools.ts

```diff
@@ -280,7 +273,9 @@ export class FluidDevtools
 		} else {
 			containerDevtools.dispose();
 			this.containers.delete(containerId);
-			this.emit("containerDevtoolsClosed", containerId);
+
+			// Post message for container list change
+			this.postContainerList();
```

Comment:
Not opposed to sending the message... but a closed container potentially still shows up in the UI menu so posting here might make no difference?

Reply:
This path isn't for closing the underlying Container, it's for closing the devtools instance associated with a specific Container. Previously, our handler for `containerDevtoolsClosed` was posting this, so this isn't a change in behavior.